Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeServerCertificates.ps1

```diff
@@ -59,7 +59,7 @@ function Get-ExchangeServerCertificates {
         # These certificates can be deleted. The Azure VM Agent re-creates certificates if needed.
         # https://docs.microsoft.com/azure/virtual-machines/extensions/features-windows
         $certificatesToExclude = @(
-            NewCertificateExclusionEntry "CN=MS-Organization-P2P-Access \[[12][0-9]{3}\]$" $true
+            NewCertificateExclusionEntry "CN=MS-Organization-P2P-Access \[[12][0-9]{3}\]$" $false
```

Comment:
Did this all of a sudden change?

Reply:
I assume that it has not yet worked. Just found the certificate on my local machine and it is indeed not self-signed. The fix should do the job of properly detecting and excluding the certificate.